app/views/users/show.html.erb

```diff
@@ -1,3 +1,18 @@
-<!-- Placeholder for user details -->
-<h1>User Details</h1>
-<p>This is where user details will be displayed.</p>
\ No newline at end of file
+<%# https://users/745 %>
+
+<img src="<%= @user.photo %>" alt="picture of user" style="width: 50px; height: 50px;">
+<h1><%= @user.name %></h1>
+Number of posts: <%= @user.posts_counter %>
+<p>Bio <br><br><%= @user.bio %></p>
+
+<ul>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <li>
+      Post #<%= index + 1 %> <br><br>
+      <%# <%= link_to post.title, user_post_path(@user, post) %>
```

Comment:
- [ ] [Optional] [Highly recommended] Please remove this commented (unused) code in line 12 because it's considered good practice to have a readable code that's clean of any commented code.

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️